packages/web-components/fast-components/src/avatar/index.ts

```diff
@@ -20,7 +20,7 @@ export class Avatar extends FoundationAvatar {
      * HTML Attribute: src
      */
     @attr({ attribute: "src" })
-    public imgSrc: string;
+    public imgSrc: string = "";
```

Comment:
Should this perhaps be null or undefined?

Reply:
Yeah good call, it should be consistent with the remaining of the PR :+1: